packages/node/src/integrations/http.ts

```diff
@@ -88,23 +88,22 @@ const _httpIntegration = ((options: HttpOptions = {}) => {
             return false;
           },
 
-          requireParentforOutgoingSpans: true,
           requireParentforIncomingSpans: false,
           requestHook: (span, req) => {
             addOriginToSpan(span, 'auto.http.otel.http');
 
+            const scopes = getCapturedScopesOnSpan(span);
+
+            const isolationScope = (scopes.isolationScope || getIsolationScope()).clone();
+            const scope = scopes.scope || getCurrentScope();
```

Comment:
Why are we moving this above the early return? As far as I can see it doesn't do anything right? (Also the "Update the isolation scope, isolate this request" comment is a bit orphanded now)

Reply:
good point, I moved stuff around multiple times here, will move it back down!